CocoaPods/iphoneos-10.0/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module CSQLite [system] {
+    header "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS10.0.sdk/usr/include/sqlite3.h"
```

Comment:
This hard-coding is required for CocoaPods. If you include SQLite.swift as a sub-project or via Carthage it uses an SDK-relative path instead.

I've raised this issue with CocoaPods in the past: https://github.com/CocoaPods/CocoaPods/issues/3942

We're always open to alternative solutions for CocoaPods users, though, and would happily take contributions that solve the problem more elegantly for them!